app/views/articles/show.html.erb

```diff
@@ -99,8 +100,8 @@
             <a class="crayons-article__cover" href="<%= cloud_cover_url(@article.main_image) %>">
               <img
                 src="<%= cloud_cover_url(@article.main_image) %>"
-                style="aspect-ratio: auto 1000 / <%= @article.main_image_height%>;"
-                width="1000" height="<%= @article.main_image_height%>"
+                style="aspect-ratio: auto 1000 / <%= @article.main_image_height %>;"
+                width="1000" height="<%= @article.main_image_height %>"
```

Comment:
I guess these are technically unrelated changes, and since they're whitespace-only, I'd  propose amending this commit to remove these whitespace changes (leaving behind only the 1-liner on L94 as the body of this PR).

Reply:
@klardotsh these changes are added by the commit hook linter. But i will check again and try to remove this.

Reply:
Oh I see. If that's the case, go ahead and leave them in then, that's fine! Thanks for using the linter!

Reply:
Thanks @klardotsh